etcd/client.go

```diff
@@ -12,7 +12,10 @@ import (
 	"net/url"
 	"os"
 	"path"
+	"strings"
 	"time"
+
+	"github.com/coreos/etcd/etcdserver/etcdhttp/httptypes"
```

Comment:
i do not want to bring in etcd's dependency into go-etcd, given the fact that we will move the client into etcd soon.